github-dark.css

```diff
@@ -1091,7 +1090,9 @@
   table.files td.message a:hover, .link-gray:hover, .news .link-gray-dark:hover,
   .news a.link-gray-dark:hover, .reblame-link:hover,
   .orgs-help-item-octicon .octicon, span.btn-link, .btn-link, a.btn-link,
-  .merge-status-list .status-actions:hover {
+  .merge-status-list .status-actions:hover,
+  .tree-browser tr.navigation-focus td .octicon-chevron-right,
+  .tree-browser tr[aria-selected="true"] td .octicon-chevron-right, {
     color: /*[[base-color]]*/ #4183c4 !important;
```

Comment:
Was adding the extra `td` necessary?

Reply:
If you look at the default css this is how it is, so I suspect this didnt work

Reply:
It wasn't working because they switched everything over to using `[aria-selected="true"]`. We're leaving in the `.navigation-focus` for GitHub Enterprise users since they lag behind in changes.